Mathlib/LinearAlgebra/Alternating.lean

```diff
@@ -103,8 +103,11 @@ instance funLike : FunLike (AlternatingMap R M N ι) (ι → M) (fun _ => N) whe
   coe_injective' := fun f g h ↦ by
     rcases f with ⟨⟨_, _, _⟩, _⟩
     rcases g with ⟨⟨_, _, _⟩, _⟩
-    cases h
-    rfl
+    congr
+
+-- Porting note: The shortcut instance is removed in Lean4.
+--  instance coeFun : CoeFun (AlternatingMap R M N ι) fun _ => (ι → M) → N :=
+--   ⟨FunLike.coe⟩
```

Comment:
Is there any reason to remove it?

Reply:
According to the porting note on `Mathlib.Algebra.Hom.Ring`, the shortcut instance is unhelpful in Lean4:
```lean
-- Porting note:
-- These helper instances are unhelpful in Lean 4, so omitting:
-- /-- Helper instance for when there's too many metavariables to apply `fun_like.has_coe_to_fun`
-- directly.
-- -/
-- instance : CoeFun (α →+* β) fun _ => α → β :=
--   ⟨RingHom.toFun⟩
```

Reply:
They're only actively bad if they mention `toFun` like that one does. This one uses `FunLike.coe` so is harmless.

Perhaps worth checking on Zulip whether we view these as phones.

Reply:
OK, I add this instance.